docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -280,11 +275,16 @@ separately.
 `cluster.remote.<cluster_alias>.transport.ping_schedule`::
 
   Sets the time interval between regular application-level ping messages that
-  are sent to ensure that transport connections to nodes belonging to remote
-  clusters are kept alive. If set to `-1`, application-level ping messages to
-  this remote cluster are not sent. If unset, application-level ping messages
-  are sent according to the global `transport.ping_schedule` setting, which
-  defaults to `-1` meaning that pings are not sent.
+  are sent to try and keep remote cluster connections alive. If set to `-1`,
+  application-level ping messages to this remote cluster are not sent. If
+  unset, application-level ping messages are sent according to the global
+  `transport.ping_schedule` setting, which defaults to `-1` meaning that pings
+  are not sent. It is preferable to correctly configure TCP keep-alives instead
+  of configuring a `ping_schedule`, because TCP keep-alives are handled by the
+  operating system and not by {es}. By default {es} sends TCP keep-alives on
```

Comment:
Nit: Maybe "By default {es} enables TCP keep-alives on". "Sends" suggests to me that Elasticsearch ultimately controls the sending.

Reply:
:+1: good call, I pushed a9a7289a643.